README.md

```diff
@@ -408,8 +414,8 @@ See: (via [GitHub Action(s)](.github/workflows/); i.e. `deploy_test_service.yml`
 
 | Term          | Definition (_we_ use)                                                                                        |
 | ------------- | ------------------------------------------------------------------------------------------------------------ |
-| `version`     | A 'number' specified in the [`SemVer`](https://semver.org/spec/v2.0.0.html)-format: `1.1.0`                  |
```

Comment:
I guess you still want to update this to YY.MM-MINOR (also at some other places)